api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -15,9 +15,7 @@
     If you intend for a changeset to be backported to older releases, you must put such a changeset in the
     liquibase-update-to-2.0.xml, which have been used for older releases.
 -->
-<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
```

Comment:
I would not expect any changes around this area.

Reply:
Okay. I'm on it